NitroxClient/GameLogic/Spawning/Metadata/PrecursorTeleporterMetadataProcessor.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+using NitroxModel.Logger;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.Spawning.Metadata
+{
+    public class PrecursorTeleporterMetadataProcessor : GenericEntityMetadataProcessor<PrecursorTeleporterMetadata>
+    {
+        public override void ProcessMetadata(GameObject gameObject, PrecursorTeleporterMetadata metadata)
+        {
+            Log.Info($"Received precursor teleporter metadata change for {gameObject.name} with data of {metadata}");
```

Comment:
Maybe Log.Debug? The `PrecursorTeleporterMetadata`-parameter seems more like debug but idk....

Reply:
This change can be made but its simular to the KeypadMetadataProcessor sunrunner created, so was just following that convention